compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -90,6 +90,7 @@ namespace TR { class Register; }
 #define IA32OpProp1_SetsCCForCompare          0x00000010
 #define IA32OpProp1_SetsCCForTest             0x00000020
 #define IA32OpProp1_SupportsLockPrefix        0x00000040
+#define IA32OpProp1_SIMDUnary                 0x00000080
```

Comment:
When does a "SIMDUnary" property apply to an opcode?  Most of the opcodes you've applied this to would not be considered "unary" operations in the traditional sense (e.g., loads and stores).  I'm worried either the name is confusing or is being applied too broadly.  Can you comment on when you expect it to apply (and perhaps add that documentation as a comment here)?

Reply:
Perhaps it can be renamed, the point was to indicate that opcode cannot have a second source operand. Many of the opcodes binary opcodes are used with only two operands, but when they are VEX/EVEX encoded, the target becomes the second source. However, the instruction is illegal, if a second source is specified for the EVEX encoding variants when there shouldn't be one.